talknet/demoTalkNet.py

```
@@ -1,726 +1,726 @@
+		image = frame
+		for face in faces[fidx]:
+			clr = colorDict[int((face['score'] >= 0))]
+			txt = round(face['score'], 1)
+			cv2.rectangle(image, (int(face['x']-face['s']), int(face['y']-face['s'])), (int(face['x']+face['s']), int(face['y']+face['s'])),(0,clr,255-clr),10)
+			cv2.putText(image,'%s'%(txt), (int(face['x']-face['s']), int(face['y']-face['s'])), cv2.FONT_HERSHEY_SIMPLEX, 1.5, (0,clr,255-clr),5)
+		vOut.write(image)
+	vOut.release()
+	command = ("ffmpeg -y -i %s -i %s -threads %d %s" % \
+		(os.path.join(pyaviPath, 'video_only.avi'), os.path.join(pyaviPath, 'audio.wav'), \
+		data_loader_thread, os.path.join(pyaviPath,'video_out.mp4')))
+	output = subprocess.call(command, shell=True, stdout=None)
```

Ruff (0.13.3)

Local variable `output` is assigned to but never used

Remove assignment to unused variable `output`

(F841)